database/index.js

```diff
@@ -1,115 +1,51 @@
-const Sequelize = require('sequelize');
-
-const sequelize = new Sequelize('postgres://localhost:5432/wawa_melon');
-
-sequelize
-  .authenticate()
-  .then(() => {
-    console.log('Connection has been established successfully.');
-  })
-  .catch((err) => {
-    console.error('Unable to connect to the database:', err);
-  });
-
-const { Model } = Sequelize;
-
-class Item extends Model {}
-Item.init({
-  id: {
-    type: Sequelize.INTEGER,
-    primaryKey: true,
-    autoIncrement: true,
-  },
-  name: {
-    type: Sequelize.STRING,
-    allowNull: false,
+const knex = require('knex')({
```

Comment:
are we allowed to use knex? I thought I read something from Robin to the contrary.